cmd/grumble/server.go

```diff
@@ -1461,7 +1504,12 @@ func (server *Server) Stop() (err error) {
 		client.Disconnect()
 	}
 
-	// Close the TLS listener and the TCP listener
+	// Wait for the HTTP server to shutdown gracefully
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(15*time.Second))
```

Comment:
Why the 15 second timeout? Did you encounter a problem? Why not just `Shutdown(ctx.Background())`?
I don't mind it, but it needs a reason -- and if there is a reason, it needs a comment.